chainer/functions/normalization/l2_normalization.py

```diff
@@ -98,7 +98,7 @@ def normalize(x, eps=1e-5, axis=1):
 
     Args:
         x (:class:`~chainer.Variable` or :ref:`ndarray`):
-            Two dimensional output variable. The first
+            `N` dimensional output variable. The first
```

Comment:
- This `` `N` `` is irrelevant to `(N, K)` above.
- It's still unclear about that the default is `axis=1` rather than `axis=tuple(range(1, N))`.

Reply:
@toslunar Thank you for the review.

I modified the docstring, and comparison is https://github.com/chainer/chainer/compare/master...crcrpar:l2-normalize-docs?expand=1.

Can I get your thoughts on this?

Reply:
LGTM except for the fix to the summary line.  The summary should be a line (https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings).  I think rephrasing "L2 norm" precedes explaining `axis`, which users could guess from the signature of the function.

Reply:
> I think rephrasing "L2 norm" precedes explaining `axis`

I didn't get the meaning of this clearly, though, did you mean one of the below two is enough?
- Normalize input by L2 norm over specified axis or axes.
- Normalize input by Euclidean norm over specified axis or axes.

Reply:
I intended to mean:
- "Normalize input by its L2 norm" is sufficient, and
- the original "Normalize input by its L2 norm (a.k.a. Euclidean norm)" is better than "Normalize input by L2 norm over specified axes".

Reply:
Thank you. I created #7371.